### ISSUE TITLE
pull request - PulseAudio class - modifier modification

### ISSUE THREAD

**Issue by jkody** [PhilippMundhenk/PulseAudioJava#2]
Hi :)

Nice wrapper project.
I am testing the PulseAudioJava library in frames of my project. The method **checkAccessTime** you provide in your library which is doing the pause between each pulseaudio call is private only; The method is pretty good so I'd like to use it directly. So I modified its modifier to **public** to have the direct access;  For now that is the  only pull request I do; So please accept it. The modification is too tiny to move version so I leave it as it is...
I am opened for discussion about the provided modification etc so please contact me if you need some more information.

P.S. And, yes, it would be nice to have the PulseAudioJava published on maven me be able to have it as a dependency a bit later.

Thanks

**Comment by PhilippMundhenk** [comment]
Whoops, never closed this after #3 was closed. I think 5 years later, it is time to close...
